src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeIndustrialGreenhouse.java

```diff
@@ -823,19 +820,25 @@ protected Widget createCropsContainerWidget(EntityPlayer player) {
                 if (clickData.shift) {
                     if (player.inventory.addItemStackToInventory(removed.input)) {
                         player.inventoryContainer.detectAndSendChanges();
-                        return;
+                    } else {
+                        mStorage.add(realID, removed);
```

Comment:
This created a small edge case dupe bug in normal crops mode.
When you have space in your inventory to get partial stack of the slot, it will give you the partial amount and add the slot back (cached outputs unchanged), then you can put them back in and repeat.